### PR TITLE
spelling fix: softwares to software

### PR DESCRIPTION
**Pull request by viridispirus**
For now I fixed all spelling mistakes with "softwares".
#137 
@jGleitz  I'm not sure if you meant to change spellings like "software's" also.
If yes I will change them too.

### CODE REVIEW

**Comment by jGleitz** [comment]
Were that really all occurrences?

>  I'm not sure if you meant to change spellings like "software's" also.

“software’s” can be correct by principle, but “a software’s” cannot.

File: doc/Requirements Specification/Parts/Application.lyx [hunk 1]
```diff
@@ -355,7 +352,7 @@ name "/A40/"
 
 \end_inset
 
- Beagle can be used to verify a software's design and implementation.
+ Beagle can be used to verify design and implementation of a software.
```
Comment (acBerger) [inline doc/Requirements Specification/Parts/Application.lyx]: This is not correct either ;)
you just changed the order of the words and kept the error.
I recommend to keep the first version and just delete the "a" ("Beagle can be used to verify a software's design and implementation.")

Comment (viridispirus) [inline doc/Requirements Specification/Parts/Application.lyx]: I thought about it, but was not sure :) Ok, I fix it again.

**Comment by acBerger** [comment]
both changes look good to me.

But I cannot say if there are more.

**Comment by viridispirus** [comment]
I reviewed the hole SRS twice.

**Comment by jGleitz** [comment]
lgtm

**Pull request merged**